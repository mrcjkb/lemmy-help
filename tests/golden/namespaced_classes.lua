---@class lz.n.KeysBase: vim.keymap.set.Opts
---@field desc? string
---@field noremap? boolean
---@field remap? boolean
---@field expr? boolean
---@field nowait? boolean
---@field ft? string|string[]

---@class lz.n.KeysSpec: lz.n.KeysBase
---@field [1] string lhs
---@field [2]? string|fun()|false rhs
---@field mode? string|string[]

local U = {}
return U
