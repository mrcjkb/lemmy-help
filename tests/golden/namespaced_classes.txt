lz.n.KeysBase : vim.keymap.set.Opts                              *lz.n.KeysBase*

    Fields: ~
        {desc?}     (string)
        {noremap?}  (boolean)
        {remap?}    (boolean)
        {expr?}     (boolean)
        {nowait?}   (boolean)
        {ft?}       (string|string[])


lz.n.KeysSpec : lz.n.KeysBase                                    *lz.n.KeysSpec*

    Fields: ~
        {1}      (string)              lhs
        {2?}     (string|fun()|false)  rhs
        {mode?}  (string|string[])



